pagopa/io-app-design-system
# [IOAPPX-352] Add the new IO typeface

> [!note]
> This PR depends on:
> * https://github.com/pagopa/io-app-design-system/pull/312

## Short description
This PR adds the new IO typeface (temporarily named **Titillio**). This change is visible only if experimental DS is on.

## List of changes proposed in this pull request
- Add the new `Titillio` typeface and relative font files
- Remove references to `ReadexPro`

### Preview

https://github.com/user-attachments/assets/eb1820dc-1f82-49e9-9218-1a2430ed6477



## How to test
1. Run the example app
2. Enable experimental DS and check every screen
